BossRush.csproj

```diff
@@ -16,7 +16,12 @@
     <None Remove="Contents\Artifacts\TokenOfGreedArtifact.png" />
     <None Remove="Contents\Artifacts\TokenofPrideArtifact.png" />
     <None Remove="Contents\Artifacts\VampirismCrystalArtifact.png" />
+    <None Remove="Contents\Items\TransmuteTablet.png" />
+    <None Remove="Localization\en-US_Mods.BossRush.ArmorSet.hjson" />
     <None Remove="Localization\en-US_Mods.BossRush.Artifacts.hjson" />
+    <None Remove="Localization\en-US_Mods.BossRush.ModEnchantment.hjson" />
+    <None Remove="Localization\en-US_Mods.BossRush.ModPerk.hjson" />
+    <None Remove="Texture\EmptyCard.png" />
```

Comment:
Why all of this exists in first place

Reply:
i don't even know, it is csproj so we probably shouldn't meddle with it unless it is required